assets/js/modules/analytics/components/settings/GA4SettingsControls.js

```diff
@@ -200,7 +204,7 @@ export default function GA4SettingsControls( { hasModuleAccess } ) {
 							}
 						/>
 
-						{ module &&
+						{ isModuleConnected &&
```

Comment:
Almost there. We actually want to show the tooltip when GA4 is **not connected**.

```suggestion
						{ ! isModuleConnected &&
```